storage/postgres/testdb/testdb.go

```diff
@@ -120,3 +120,13 @@ func sanitize(script string) string {
 func getConnStr(name string) string {
 	return fmt.Sprintf("database=%s %s", name, *pgOpts)
 }
+
+// OpenTestDBOrDie attempts to return a connection to a new postgres
+// test database and fails if unable to do so.
+func OpenTestDBOrDie(ctx context.Context) *sql.DB {
```

Comment:
nit: The convention is to suffix "OrDie" on the end of the function you're calling, i.e. "NewTrillianDBOrDie"

Reply:
For clarification, do you mean `NewTrillianDBOrDie` should just panic if there's an error? If that's the case we don't really need the `OpenTestDB` function. I'm going to go ahead with that assumption but let me know if that's wrong.

Reply:
I just meant you should rename this function. There isn't already a `NewTrillianDBOrDie` function, is there?

Reply:
gotcha, will do